packages/graph/hash_graph/lib/graph/src/api/rest/data_type.rs

```diff
@@ -272,8 +272,8 @@ struct UpdateDataTypeRequest {
     request_body = UpdateDataTypeRequest,
 )]
 async fn update_data_type<P: StorePool + Send>(
-    body: Json<UpdateDataTypeRequest>,
     pool: Extension<Arc<P>>,
+    body: Json<UpdateDataTypeRequest>,
```

Comment:
Question: Was this a recommended change, or a compile error with 0.6?  :)

Reply:
It's a compile error since 0.6 to not pass the body as the last parameter to avoid having multiple bodies in the signature.